### PR TITLE
Modal 

### PR DESCRIPTION
**Pull request by Rdecoster**
Modal is now works mostly correctly. 
Needs css fixd.
Made changes where needed for style on modal.

### CODE REVIEW

**Review by carina-lxx — APPROVED** [review]

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -22,6 +24,13 @@ background-color: #ffffff;
 font-family: "Times New Roman", Times, serif;
```
Comment (j-scola) [inline client/src/components/App.jsx:24]: is there times new roman in this widget?

File: client/src/components/Modal/GalSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    this.state = {
+      value: '',
+      currentSlide: 1,
+      imageURL: [],
+    };
+    this.incromentCounter = this.incromentCounter.bind(this);
+    this.decromentCounter = this.decromentCounter.bind(this);
+  }
+
+  incromentCounter(event) {
```
Comment (j-scola) [inline client/src/components/Modal/GalSlider.jsx:46]: spelling for decrement and increment is off here, not that important to fix but worth pointing out in a review

File: client/src/components/Modal/SliderContent.jsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  0%   { opacity: 0; }
+  11.11%   { opacity: 1; }
+  33.33%  { opacity: 1; }
+  44.44%  { opacity: 0; }
+  100% { opacity: 0; }
+}
+
+img{
+    opacity:0;
+    animation-name: fade;
```
Comment (j-scola) [inline client/src/components/Modal/SliderContent.jsx:32]: awesome!

File: client/src/components/Modal/SliderContent.jsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+      </ImageContainer>
+      <Arrows direction onClick={incromentCounter}>
+        {' '}
+        <img src="https://img.icons8.com/material/24/000000/chevron-right--v1.png" />
+      </Arrows>
+    </dic>
+  );
+}
+
+export default SliderContent;
```
Comment (j-scola) [inline client/src/components/Modal/SliderContent.jsx:151]: lots of good work on this one!

**Review by j-scola — COMMENTED** [review]
nice work!

File: client/src/components/App.jsx (hunk 1 above)
Comment (Rdecoster) [reply client/src/components/App.jsx:24]: Yes there is. I was surprised

**Pull request closed without merging**